layers/org/packages.el

```diff
@@ -388,3 +389,7 @@ Will work on both org-mode and any mode that accepts plain html."
 (defun org/init-htmlize ()
  (use-package htmlize
     :defer t))
+
+(defun org/init-orgit ()
+  (use-package orgit
+    :defer t))
```

Comment:
Do you experienced any troubles? In my installation all works fine.

I guess it's loaded by org package, like htmlize package. It's doesn't
have :init block too.

Sylvain Benner writes:

>> @@ -388,3 +389,7 @@ Will work on both org-mode and any mode that accepts plain html."
>>  (defun org/init-htmlize ()
>>   (use-package htmlize
>>      :defer t))
>> +
>> +(defun org/init-orgit ()
>> +  (use-package orgit
>> +    :defer t))
>
> How the package will be loaded ? I guess you are missing a hook in a `:init` block.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/syl20bnr/spacemacs/pull/3991/files#r47055284

--
best regards,
Skorzhinskiy Mikhail